lib/pronto/blacklist.rb

```diff
@@ -32,6 +32,10 @@ def match?(line, blacklist_string)
         return false if PathSpec.new(options['exclude']).match(line.patch.new_file_full_path)
       end
 
+      # if we should check comments for occurance of blacklisted string
+      if !evaluate_comments?(options)
+        return false if line.strip.start_with?("#")
```

Comment:
This only works for comments that are on their own line.

Reply:
This is what rubocop uses behind the scenes to figure out if something is a comment
https://github.com/whitequark/parser
https://github.com/whitequark/parser/blob/master/lib/parser/source/comment.rb
https://github.com/rubocop-hq/rubocop/blob/841b6fd71c65c3d6692dbdf4f866d1c62249e21d/lib/rubocop/processed_source.rb#L154